webroot/src/components/Lists/ListContainer/ListContainer.spec.ts

```diff
@@ -0,0 +1,157 @@
+//
+//  ListContainer.spec.ts
+//  InspiringApps modules
+//
+//  Created by InspiringApps on 5/27/2020.
+//  Copyright © 2024. InspiringApps. All rights reserved.
+//
+
+import { expect } from 'chai';
+import { mountShallow } from '@tests/helpers/setup';
+import ListContainer from '@components/Lists/ListContainer/ListContainer.vue';
+import CompactToggle from '@components/Lists/CompactToggle/CompactToggle.vue';
+import Sorting from '@components/Lists/Sorting/Sorting.vue';
+import Pagination from '@components/Lists/Pagination/Pagination.vue';
+
+describe('ListContainer component', async () => {
+    it('should mount the component', async () => {
+        const wrapper = await mountShallow(ListContainer, {
+            props: {
+                list: [],
+                paginationId: 'test',
+                // pageChange: () => null,
```

Comment:
Each test spells out the possible input props. For us, having the full set available speeds up the creation of new tests & also helps update / troubleshoot existing tests. We've considered creating some type of dynamic factory function (something we would favor in app code), but for tests we felt like it would just be an extra piece of logic that could break. Not sure how you feel about that.

Reply:
Understood—thanks for explaining!